cli/command_snapshot_create.go

```diff
@@ -105,6 +112,21 @@ func runSnapshotCommand(ctx context.Context, rep repo.RepositoryWriter) error {
 	return errors.Errorf("encountered %v errors:\n%v", len(finalErrors), strings.Join(finalErrors, "\n"))
 }
 
+func getTags() (map[string]string, error) {
+	tags := map[string]string{}
+
+	for _, tagkv := range *snapshotCreateTags {
+		tagFields := strings.Split(tagkv, ":")
```

Comment:
Please address upstream comment https://github.com/kopia/kopia/pull/956#discussion_r610956427

It allows having tags of the form `k:v:x`

```suggestion
		tagFields := strings.SplitN(tagkv, ":", 2)
```

nit: could rename `tagFields` to `parts`